auth.php

```diff
@@ -47,86 +47,44 @@
 
 $currenturl = new moodle_url('/admin/tool/mfa/auth.php');
 
-if (isset($SESSION->tool_mfa_authenticated) && $SESSION->tool_mfa_authenticated) {
+// Check for overall state.
+$state = \tool_mfa\manager::check_status();
```

Comment:
'check_status' implies we are doing some checking, but it isn't. This is more of a 'get_status'

I'm thinking it would be clearer to have 2 methods, check_status which also does the redirect / cannot_login, and it internally calls get_status. I'm pretty sure we'll want to call check_status in a couple places